Kwenta/kwenta
# Add skew and trading history

## Description
Implement a trade history & skew panel

## Related issue
#639 

## Motivation and Context
Show the trade history and skew panel.

## How Has This Been Tested?
1. Switch to the different markets and check the skew and trade history
3. Check the layout when there is no trade

## Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/4819006/164341468-2312020c-e3aa-444b-8792-d237aabc9336.png)
![image](https://user-images.githubusercontent.com/4819006/164341514-8a0583f2-6c01-434d-9fd7-07f2af42f83d.png)
![image](https://user-images.githubusercontent.com/4819006/164342747-32990f77-71a3-4129-bb12-037292bf6180.png)
